packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsEmailsBlocklistSection.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+import { v4 } from 'uuid';
+
+import { SettingsAccountsEmailsBlocklistInput } from '@/settings/accounts/components/SettingsAccountsEmailsBlocklistInput';
+import { SettingsAccountsEmailsBlocklistTable } from '@/settings/accounts/components/SettingsAccountsEmailsBlocklistTable';
+import { H2Title } from '@/ui/display/typography/components/H2Title';
+import { Section } from '@/ui/layout/section/components/Section';
+import { mockedBlockedEmailList } from '~/testing/mock-data/accounts';
+
+export const SettingsAccountsEmailsBlocklistSection = () => {
+  const [blockedEmailList, setBlockedEmailList] = useState(
+    mockedBlockedEmailList,
+  );
+
+  const handleBlockedEmailRemove = (id: string) =>
+    setBlockedEmailList((previousBlockedEmailList) =>
+      previousBlockedEmailList.filter((blockedEmail) => blockedEmail.id !== id),
+    );
+
+  const updateBlockedEmailList = (email: string) =>
+    setBlockedEmailList((prevState) => [
+      ...prevState,
+      { id: v4(), email: email, blocked_at: '21/12/2023' },
+    ]);
```

Comment:
Later we will need to check if the value `email` is indeed an email and we will also need to check if the email isn't already blocked.
I think that even if these are mock data, we could put the current date in `blocked_at`

Reply:
Yeah, we can put the check later on once we integrate with the backend, also I didn't find any error state design for this in Figma.